src/bun.js/bindings/webcrypto/CryptoKeyAES.cpp

```diff
@@ -25,7 +25,7 @@
 
 #include "config.h"
 #include "CryptoKeyAES.h"
-
+#include "../wtf-bindings.h"
```

Comment:
did you mean to update `simdutf` above?

Reply:
just to use `Bun::base64URLEncodeToString(m_key);`

Reply:
not intended was resulted from `make fmt-cpp`

Reply:
oh we need to undo that

it will break stuff. can you revert the changes to simdutf?

Reply:
Reverted